src/foam/nanos/cron/SimpleIntervalScheduleView.js

```diff
@@ -43,9 +43,9 @@ foam.CLASS({
             })
           }
         ).addClass(this.myClass('container'))
-        .start(data.START_DATE.__).addClass(this.myClass('fullWidth')).end()
-        .start(data.REPEAT.__).addClass(this.myClass('halfWidth')).end()
+        .start(data.START_DATE.__).addClass(this.myClass('halfWidth')).end()
         .start(data.FREQUENCY.__).addClass(this.myClass('halfWidth')).end()
+        .start(data.REPEAT.__).addClass(this.myClass('halfWidth')).end()
```

Comment:
Was there a reason for flipping repeat after frequency?

Reply:
@tharmaman I followed the attached design in the ticket. Would you like me to undo this?